napari/utils/_proxies.py

```diff
@@ -70,7 +78,11 @@ def __dir__(self):
         return [x for x in dir(self.__wrapped__) if not _SUNDER.match(x)]
 
     @classmethod
-    def create(cls, obj: Any) -> 'PublicOnlyProxy':
+    def create(cls, obj: Any) -> Union['PublicOnlyProxy', Any]:
+        # restrict the scope of this proxy to napari objects
+        if not getattr(type(obj), '__module__', '').startswith(cls.__root):
```

Comment:
why `type` here?

Reply:
good question.  I guess this would actually _fail_ to wrap certain things that should be wrapped huh?  i can make it just `obj` ...  are you thinking of a specific case?  or just curious